kinto_admin/tests/test_plugin_setup.py

```diff
@@ -0,0 +1,18 @@
+from kinto_admin import __version__ as admin_version
+from . import BaseWebTest
+from kinto.tests.support import unittest
+
+
+class HelloViewTest(BaseWebTest, unittest.TestCase):
+
+    def test_capability_is_exposed(self):
+        self.maxDiff = None
```

Comment:
No that's something we want to keep here to see the full diff in case this test fails.